### PR TITLE
Feat/deliverable3

### PR DESCRIPTION
**Pull request by tomas10100**
Deliverable3

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,18 +1,11 @@
...
-
-  const handleNavigate = (band) => {
-    setIsHome(band);
-  };
-
   return (
-    <PageTemplate handleNavigate={handleNavigate}>
-      {isHome ? <Home handleNavigate={handleNavigate} /> : <ProductList />}
-    </PageTemplate>
+    <NavbarState>
```
Comment (franciscogranadoswizeline) [inline src/App.js:6]: is state for the navbar of for the app? the name does not match with the component inside the wrapper

File: src/components/layout/Header/Header.js [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
-const Header = ({ handleNavigate }) => {
+const Header = () => {
   return (
     <Heading>
-      <Logo onClick={() => handleNavigate(true)} />
+      <Logo />
       <ActionsContainer>
         <NavDeskContainer>
-          <NavbarDesktop onClick={() => handleNavigate(true)} />
+          <NavbarDesktop />
```
Comment (franciscogranadoswizeline) [inline src/components/layout/Header/Header.js:12]: what props does NavbarDesktop has? in this path "./common/NavbarDesktop/NavbarDesktop.js" the component has props but what values are? why this component does not get any of them?

File: src/components/product/Gallery/Gallery.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  Image,
+} from "./styled";
+import PropTypes from "prop-types";
+
+const Gallery = ({ gallery }) => {
+  const images = createGalleryAdapter(gallery);
+  const [currentImage, setCurrentImage] = useState(0);
+  let max = images.length - 1 ?? 0;
+
+  const handleMainImage = (url) => {
```
Comment (franciscogranadoswizeline) [inline src/components/product/Gallery/Gallery.js:18]: why not just setCurrentImage?

File: src/pages/SearchResultsPage/index.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+  return (
+    <Container>
+      <SectionHeading style={{ marginTop: "30px" }}>
+        Search Results:
+      </SectionHeading>
+      {isLoading ? <SpinnerBounce /> : <Grid productsList={searchResults} />}
+      <PageControlls
+        setCurrentPage={setCurrentPage}
+        pages={searchResults.totalPages ?? 0}
```
Comment (franciscogranadoswizeline) [inline src/pages/SearchResultsPage/index.js:23]: if this condition does not apply what is the default value?

**Review by franciscogranadoswizeline — COMMENTED** [review]
check the comments please. Good job.

File: src/components/product/Gallery/Gallery.js (hunk 3 above)
Comment (tomas10100) [reply src/components/product/Gallery/Gallery.js:18]: I was looking for make more reusable this function but no happen, is a good idea only use the setter

File: src/App.js (hunk 1 above)
Comment (tomas10100) [reply src/App.js:6]: Yes, you're right, is wrong, this state is for the Header component is to managment the open/close the mobile nav but still  I'm not done implementing it yet, I fix it in the new branch

File: src/components/layout/Header/Header.js (hunk 2 above)
Comment (tomas10100) [reply src/components/layout/Header/Header.js:12]: the implementation is wrong, i fix it

File: src/pages/SearchResultsPage/index.js (hunk 4 above)
Comment (tomas10100) [reply src/pages/SearchResultsPage/index.js:23]: inside the Grid component, if fail render a message, I check it

**Pull request merged**